db/blip_handler.go

```diff
@@ -136,7 +148,11 @@ func collectionBlipHandler(next blipHandlerFunc) blipHandlerFunc {
 	return func(bh *blipHandler, bm *blip.Message) error {
 		collectionIndexStr, ok := bm.Properties[BlipCollection]
 		if !ok {
-			bh.collection = bh.db
+			bh.collection = &DatabaseCollectionWithUser{
```

Comment:
If this is a collection-scoped handler (collectionBlipHandler), should we always expect a collection property?  Otherwise I'm not sure what specific collection we'd be falling back to here.